var/spack/repos/builtin/packages/axom/package.py

```diff
@@ -37,6 +37,8 @@ class Axom(CachedCMakePackage, CudaPackage):
     homepage = "https://github.com/LLNL/axom"
     git      = "https://github.com/LLNL/axom.git"
 
+    tags = ['ecp', 'e4s']
```

Comment:
Note this appears in slides but is not currently listed as an ST product.

Reply:
Definitely part of E4S. Not sure about ECP

Reply:
okay i removed `ecp` tag from axom package